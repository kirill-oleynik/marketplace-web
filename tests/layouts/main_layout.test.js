const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const MainLayout = require('../../app/layouts/main_layout').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <MainLayout>
        <div>Hello</div>
      </MainLayout>
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
