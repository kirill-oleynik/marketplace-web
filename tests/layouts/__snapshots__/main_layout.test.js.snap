// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="page-container"
>
  <Connect(HeaderContainer) />
  <main
    className="flex-grow-1"
  >
    <div>
      Hello
    </div>
  </main>
  <MainFooter />
  <Component />
</div>
`;
